docker-compose.yaml

```diff
@@ -1,26 +1,58 @@
-version: '3.9.1'
+version: "3.9.1"
 
-# You must manually run `ankerctl config import` atleast once
-# docker run \
-# -v ankerctl_vol:/root/.config/ankerctl \
-# -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-# ankerctl config import /tmp/login.json
 services:
-  ankerctl:
-    image: ankerctl/ankerctl:latest
-    container_name: ankerctl
-    restart: unless-stopped
-    build: .
-    environment:
-      - FLASK_PORT=4470
-      - FLASK_HOST=0.0.0.0
-    volumes:
-      - ankerctl_vol:/root/.config/ankerctl
-    ports:
-      - 127.0.0.1:4470:4470
-    entrypoint: "/app/ankerctl.py"
-    command: ["webserver", "run"]
+    ankerctl_seed_mac:
+        image: busybox
+        container_name: ankerctl_seed_mac
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            - "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
+        command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"
+    ankerctl_seed_linux:
```

Comment:
There is no version of AnkerMake for Linux just yet, and if there is in the future the path will look more like `$HOME/.local/share/<AnkerMakeSpecificPath>`

Reply:
INFORMATIONAL: Not for v1.0.0 ship but for v.1.1 we can look at their alpha build of the new slicer since that does have a linux build.

Reply:
Ok after futzing with this for about an hour I have it working this way:

`volumes:
     - ankerctl_vol:/root/.config/ankerctl
    - $HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json 
    #command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"`
  NO command argument

Reply:
🤔 Hmm, that is interesting. If the command argument is commented out then the `cp` won't work. 

I suspect what happened is another previous attempt copied the login.json to the volume. 

If you purge your docker volumes, and then try again I'm fairly confident that this will not work with that line commented out. 

```
docker system prune
docker volume rm ankermake-m5-protocol_ankerctl_vol
docker compose up
```